crates/sui-graphql-rpc/schema.graphql

```diff
@@ -0,0 +1,823 @@
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Open Questions
+# --------------
+# - Rules for nullabilit/non-nullability
+# - Pattern for multi-gets
+# - Encoding RPC errors
+# - `type` (vs `scalar`) for Move Types
+# - Pattern for upcasting and downcasting
+
+# Check Completeness Against Existing APIs
+# -------------
+# DONE Read
+# DONE - Objects
+# DONE - Transactions
+# DONE - Checkpoints
+# DONE - Events
+# DONE - ProtocolConfig
+# DONE - Chain Identifier
+# DONE Coin
+# DONE Extended
+# DONE Governance
+# DONE Indexer
+# DONE Move Utils
+# TODO Write
+
+# DONE Port designs from Notion
+# DONE - Display
+# DONE - SuiNS
+# DONE Data Consistency API
+# TODO Flesh out Connections
+# TODO - TotalTransactionBlocks for the PTB connection
+
+schema {
+  query: Query
+  subscription: Subscription
```

Comment:
I would really prefer if we start without supporting Subscriptions. Today indexer doesn't support any subscriptions and given we'd like these services to be stateless we'd need to think carefully about how to even go about supporting subscriptions

Reply:
What will the migration path be for users who currently use event subscription? AFAIK event subscription is a pretty commonly used feature for advanced app builders.

Reply:
I've made a note that we can build subscriptions after the MVP.